Makefile

```diff
@@ -0,0 +1,17 @@
+# Building docker image
+TAG = "v1.0.0"
```

Comment:
You don't need double quotes around the value in Make variables. Is there a reason that you believe you need the quotes? As per Make documentation, everything to the right of the `=` (ignoring whitespace) becomes the value. So this would mean that the quotes are part of the value. Is this intended?

Reply:
<img width="759" alt="Screen Shot 2020-04-15 at 4 37 16 PM" src="https://user-images.githubusercontent.com/8756596/79386229-d7487400-7f37-11ea-982b-0181dcc7f3f9.png">

Assignments, with and without quotes, seem to behave the same way. However, I have removed the quotes for the sake of consistency.

Reply:
They behave the same way for how this is interpreted by the shell, yes. But you can see in your own example that the variable values are indeed different.

Reply:
Fair enough.